docs/config-file/v2.rst

```diff
@@ -175,6 +175,29 @@ Example:
   manage the build, this setting will not have any effect. Instead
   add the extra requirements to the ``environment`` file of Conda.
 
+Packages list
+'''''''''''''
+
+Install packages from a list of requirements. This allows you to embed requirements in the
+``.readthedocs.yml`` configuration directly.
+
+:Key: `packages`
+:Type: ``string``
+:Required: ``true``
+
+Example:
+
+.. code-block:: yaml
+
+   version: 2
+
+   python:
+     version: 3.7
+     install:
+       - packages:
+         - mkdocs-material
+         - Pygments
+
```

Comment:
We should add something like the packages section

`With the previous settings, Read the Docs will execute the next commands:`...